### ISSUE TITLE
关于centernet的physical structure的指标

### ISSUE THREAD

**Issue by ZHEGG** [wangwen-whu/WTW-Dataset#17]
在使用centernet的代码复现其physical structure中的P,R,F指标时，发现自己的结果为
![1](https://user-images.githubusercontent.com/65839993/180694560-c198e1e3-1ce2-41b1-b8bf-d8c3e3effde4.JPG)
在iou=0.9时检测结果为0
并可视化了一些结果，发现其大概是正常的
![2](https://user-images.githubusercontent.com/65839993/180694877-560e816e-ba1a-43e5-ad54-4f9693e7fe9a.png)
所以对您iou=0.9的指标有些疑惑
我使用了3090x4来复现您的结果其opt如下

==> torch version: 1.7.0+cu110
==> cudnn version: 8004
==> Cmd:
['src/main.py', 'ctdet', '--exp_id', 'wtw_lr_1024', '--dataset', 'wtw', '--batch_size', '32', '--master_batch', '9', '--lr', '1.25e-4', '--gpus', '0,1,2,3', '--num_epochs', '150', '--lr_step', '90,120', '--input_res', '1024']
==> Opt:
  K: 100
  aggr_weight: 0.0
  agnostic_ex: False
  arch: dla_34
  aug_ddd: 0.5
  aug_rot: 0
  batch_size: 32
  cat_spec_wh: False
  center_thresh: 0.1
  chunk_sizes: [9, 8, 8, 7]
  data_dir: /home/mdisk2/bianzhe/CenterNet/src/lib/../../data
  dataset: wtw
  debug: 0
  debug_dir: /home/mdisk2/bianzhe/CenterNet/src/lib/../../exp/ctdet/wtw_lr_1024/debug
  debugger_theme: white
  demo: 
  dense_hp: False
  dense_wh: False
  dep_weight: 1
  dim_weight: 1
  down_ratio: 4
  eval_oracle_dep: False
  eval_oracle_hm: False
  eval_oracle_hmhp: False
  eval_oracle_hp_offset: False
  eval_oracle_kps: False
  eval_oracle_offset: False
  eval_oracle_wh: False
  exp_dir: /home/mdisk2/bianzhe/CenterNet/src/lib/../../exp/ctdet
  exp_id: wtw_lr_1024
  fix_res: True
  flip: 0.5
  flip_test: False
  gpus: [0, 1, 2, 3]
  gpus_str: 0,1,2,3
  head_conv: 256
  heads: {'hm': 1, 'wh': 2, 'reg': 2}
  hide_data_time: False
  hm_hp: True
  hm_hp_weight: 1
  hm_weight: 1
  hp_weight: 1
  input_h: 1024
  input_res: 1024
  input_w: 1024
  keep_res: False
  kitti_split: 3dop
  load_model: 
  lr: 0.000125
  lr_step: [90, 120]
  master_batch_size: 9
  mean: [[[0.7733555  0.78804266 0.7961663 ]]]
  metric: loss
  mse_loss: False
  nms: False
  no_color_aug: False
  norm_wh: False
  not_cuda_benchmark: False
  not_hm_hp: False
  not_prefetch_test: False
  not_rand_crop: False
  not_reg_bbox: False
  not_reg_hp_offset: False
  not_reg_offset: False
  num_classes: 1
  num_epochs: 150
  num_iters: -1
  num_stacks: 1
  num_workers: 4
  off_weight: 1
  output_h: 256
  output_res: 256
  output_w: 256
  pad: 31
  peak_thresh: 0.2
  print_iter: 0
  rect_mask: False
  reg_bbox: True
  reg_hp_offset: True
  reg_loss: l1
  reg_offset: True
  resume: False
  root_dir: /home/mdisk2/bianzhe/CenterNet/src/lib/../..
  rot_weight: 1
  rotate: 0
  save_all: False
  save_dir: /home/mdisk2/bianzhe/CenterNet/src/lib/../../exp/ctdet/wtw_lr_1024
  scale: 0.4
  scores_thresh: 0.1
  seed: 317
  shift: 0.1
  std: [[[0.28359458 0.26819503 0.26328853]]]
  task: ctdet
  test: False
  test_scales: [1.0]
  trainval: False
  val_intervals: 5
  vis_thresh: 0.3
  wh_weight: 0.1

期待您的回复！
